src/loaders/lottie/tvgLottieBuilder.cpp

```diff
@@ -545,6 +545,7 @@ static void _updateText(LottieGroup* parent, LottieObject** child, float frameNo
 
     auto scale = doc.size * 0.01f;
     float spacing = text->spacing(frameNo) / scale;
+    float trackingOffset = doc.tracking / 1000.0f * (doc.size / scale);
```

Comment:
same logic?
```
auto tracking = doc.tracking * 0.1f
```

Reply:
If the `doc.size` is zero, Yes. My test sets also don't have that value, it works.
But i'm not sure if it could cause different result in case it has `doc.size` value.

I've never had lottie file has `doc.size` value, Have you?

Reply:
if the doc.size(font) size zero, then it could skip the text at all.
tracking doesn't need to take care of that scenario.

Reply:
if the doc.size is zero, then scale value will be 0. it would not draw anything about text. So tracking doesn't need to care about it.

Reply:
You're right, sorry never mind the comment!